api/v1/nodescan/service.go

```diff
@@ -270,6 +273,15 @@ func (s *serviceImpl) GetNodeVulnerabilities(_ context.Context, req *v1.GetNodeV
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	// Handle the new format of the request and scan node inventory additionally
+	if features.RHCOSNodeScanning.Enabled() && req.GetNodeInventory() != nil {
+		// TODO(ROX-12968): resolve hardcoded value uncertifiedRHEL
```

Comment:
I don't think that ROX-12968 is appropriate task for this.

Moreover, you'll get all necessary info in `GetNodeVulnerabilitiesRequest.notes` once you add that attribute. Therefore, this TODO can be addressed as part of this same PR.